ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -138,6 +141,23 @@ class TestCommon {
                 url(response.header("Location"))
             }.build()
         }
+
+        /**
+         * Verify that the value of NotBefore is less than the value for NotOnOrAfter.
+         */
+        fun validateTimeWindow(node: Node, samlCode: SAMLCoreRefMessage) {
```

Comment:
The motivation behind passing those Nodes is to avoid doing the null check everywhere else

Reply:
❗️ This function feels like it should live in a verification file instead of a utility file. Especially since we like to keep all of our `SAMLComplianceException`s contained within the verification package.